bin/slackin

```diff
@@ -6,7 +6,7 @@ var slackin = require('./../node').default;
 
 args
   .option(['p', 'port'], 'Port to listen on [$PORT or 3000]', require('hostenv').PORT || process.env.PORT || 3000)
-  .option(['h', 'hostname'], 'Hostname to listen on [$HOSTNAME or 0.0.0.0]', require('hostenv').HOSTNAME || process.env.WEBSITE_HOSTNAME || '0.0.0.0')
+  .option(['H', 'hostname'], 'Hostname to listen on [$HOSTNAME or 0.0.0.0]', require('hostenv').HOSTNAME || process.env.WEBSITE_HOSTNAME || '0.0.0.0')
```

Comment:
I think the hostname arg should remain as `h` (lowercase) for backwards compatibility. That's the way it was before `args` got introduced: https://github.com/rauchg/slackin/blob/4ea80c42414f346c1f22e8e0bf81b60f90c937f4/bin/slackin#L11 and changing it will probably break some people's deployments.